rs/protobuf/src/gen/p2p/registry.crypto.v1.rs

```diff
@@ -0,0 +1,210 @@
+/// A list of subnets that can sign with this ECDSA key.
```

Comment:
this seems unrelated

Reply:
The new protos use protos defined in proto `types` and it regenerates all proto dependencies. If we would put the new `Stripped` type in `def/types` we would not need to regenerate all.